seed/models/properties.py

```diff
@@ -693,12 +695,43 @@ def merge_relationships(cls, merged_state, state1, state2):
 
             # connect back up the scenario measures
             for scenario_id, measure_list in scenario_measure_map.items():
+
                 # create a new scenario from the old one
                 scenario = ScenarioClass.objects.get(pk=scenario_id)
+
+                # first get meters and meterreadings
+                meters = MeterClass.objects.filter(scenario_id=scenario_id)
+
                 scenario.pk = None
                 scenario.property_state = merged_state
                 scenario.save()  # save to get new id
 
+                # this feels wasteful but I guess it is the pattern...
+                for m in meters:
+
+                    # first get all readings
+                    readings = MeterReadingClass.objects.filter(meter_id=m.id)
+
+                    # then duplicate
+                    meter = MeterClass.objects.get(pk=m.id)
+                    meter.pk = None
+                    meter.scenario_id = scenario.id
+                    meter.save()  # save to get new id / association
+
+                    new_readings = {
```

Comment:
To keep things DRY, I'd suggest using the Meter model's `copy_readings` method here with the `overlap_possible` flag set to `False`.

https://github.com/SEED-platform/seed/blob/develop/seed/models/meters.py#L99

Reply:
done.